scripts/sql/delta/v190-v200/apolloconfigdb-v190-v200-after.sql

```diff
@@ -62,7 +62,7 @@ ALTER TABLE `Cluster`
 -- Ignore TABLE `Item`, add unique index in future
 
 ALTER TABLE `Namespace`
-    ADD UNIQUE INDEX `UK_AppId_ClusterName_NamespaceName_DeletedAt` (`AppId`(191),`ClusterName`(191),`NamespaceName`(191),`DeletedAt`),
```

Comment:
You shouldn't change this file, you should create a new sql file to do the delta changes things.

Reply:
Ok, let me modify it.